recipe/meta.yaml

```diff
@@ -42,7 +42,10 @@ outputs:
         - cmake
         - make  # [not win]
       host:
-        - tiledb =2.18.2
+        # Please see
+        # https://github.com/TileDB-Inc/tiledbsoma-feedstock/pull/76#discussion_r1457880263
+        # before changing this:
```

Comment:
I'm not sure this comment is necessary. We had to do the awkward pinning of `tiledb 2.18.2` because we hadn't updated the numpy pins, which was forcing the solver to install tiledb 2.18.2 in all the Python envs. Now that we've updated the numpy pins, this workaround is no longer needed.

Reply:
Good call -- thank you!